docker-compose/swe-d2cc/config/swe-d2cc-data-bridge-application.yml

```diff
@@ -112,10 +112,17 @@ data-bridge:
       polling-delay-in-ms: 5000
       max-messages-per-poll: 10
       max-pool-size: 2
+      timeouts:
+        data-timeout: 60000
+        default-timeout: 500
+        connect-timeout: 1000
+        session-wait-timeout: 1000
   sinks:
     minio:
       url: http://minio:9000
       access-key: gridcapa
       secret-key: gridcapa
       bucket: gridcapa
       base-directory: /SWE/D2CC
+logging.level:
```

Comment:
I believe your forgot to delete this logging.level

Reply:
true I saw it too : removed already